electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+ 
+	(d) Repeat steps 7b and 7c for M2 motor.
+	
+8. Once both motors are spinning the correct direction and the encoders respond accordingly when commanded through the PWM Signal tab, move on to the Velocity Settings tab.
+
+![](../images/calibration/is6.png)
+**Figure 6**
+
+Here we’ll be tuning the motor settings. We need to do this in order to have fluid control of the motor, and it needs to be done for each individual motor as they will all differ slightly out of the box. Therefore, each motor must be be individually tuned. For more information on motor PID controls and what they do you can visit:
+
+* [An Introduction to PID Control with DC Motor](https://medium.com/pollenrobotics/an-introduction-to-pid-control-with-dc-motor-1fa3b26ec661)
+* [The Working of a PID Controller](https://www.elprocus.com/the-working-of-a-pid-controller/)
+
+There has been significant feedback that the BasicMicro Motion Studio auto tune sometimes does not work, and so we have included a section at the end of this document about how to tune the motors yourself. 
+
+	(a) Begin by entering in the QPPS (quadrature pulses per second), or the maximum number of encoder ticks per second. To find this value, drive the motor at max speed in the PWM setting and look at the box labeled ’M1/M2 Speed’.
+
+![](../images/calibration/is7.png)
+**Figure 7**
+
+	(b) Now hit the Tune M1 button. This will populate the Velocity P, I, and D variables on the M1 side. Repeat this step for M2 motor as well.
+
+![](../images/calibration/is8.png)
+**Figure 8**
+
+	(c) Once both motors on this motor controller have been tuned go back to the General settings tab and update the Motor Parameters to have a Max current of 3.0 Amps. Then go to Device − > Save Settings and then exit. Repeat all of steps 6 through 8 for the other drive motors.
+ 	
+## 1.3 Corner Motor Calibration
+9. Perform the following for the RoboClaw addresses 131 and 132:
+
+	(a) Perform step 6 above for the corner motors, making sure to set the encoder type to absolute.
+
+	(b) Go to the PWM Settings tab and slowly drive the M1 corner forward. Verify that the encoder value goes up. If it does not, switch the motor wire leads at M1A and M1B.2
+
+	(c) Repeat step 9b for M2 motor
+
+	(d) You may need to set the absolute encoder ”zero” point in a different location. In order to avoid ”clipping” of the range of the encoder, we’ll set the mid-point of the absolute encoder to be the middle of the turning arc.
+
+		* Navigate to the PWM Settings tab to move the corner to its ”center” position, which is where the wheel is directly forward.
+		* Unscrew the small gear at the bottom of the absolute encoder
+		* Once the absolute encoder shaft is uncoupled from the robot, spin the encoderuntil you determine the max value. This should be somewhere in the range of 1400-18004.
+		* As opposed to the drive motors, the corners use absolute encoders that are defined in one direction only, so you cannot switch the direction of the encoder
```

Comment:
some content got messed up in formatting here. See original pdf. I would put the footnotes in some kind of separate box. See https://stackoverflow.com/a/58758521/4292910 for inspiration